guides/NIGHTLY_PIPELINES.md

```diff
@@ -0,0 +1,109 @@
+# Nightly Pipelines
+
+This guide outlines how to find, diagnose, take action and report on the RHMI 2.x nightly pipelines. 
+
+It assumes no prior exposure to the integreatly-operator or the pipelines.
+
+
+
+## Resources
+
+- Documentation
+
+  - [Detailed overview of RHMI 2.x nightly pipelines](https://docs.google.com/document/d/1eYtezxJHgOWENs_KRL27gp6AQmggCLUMVkIrSaNIcEQ/edit)
+
+- Pipelines
+
+  - [Master Installation](https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Nightly/job/rhmi-install-master/) 
+  - [Addon Installation](https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Nightly/job/rhmi-install-addon-flow/)
+
+- Pipeline configuration
+
+  - [Pipeline configuration repo](https://gitlab.cee.redhat.com/integreatly-qe/ci-cd) 
+
+> You will need to be on the Red Hat vpn to access Gitlab.
+> Login to Jenkins and Gitlab is with your kerberos credentials. 
+
+
+
+
+## Flaky tests
+
+Pipelines runs may fail because of test(s) that pass/ fail inconsistently, ie a test produces a different result when re-run even though nothing related to the test has changed. These tests are regarded as `flaky`. 
```

Comment:
Actually pipelines will not fail if any of the tests failed. The `tests` stage will however be marked as `UNSTABLE`. If that is the case, search for `FAIL: ` (note the `:` - this will speed up the search) inside the job output to see, which functional tests failed.